src/main/java/com/mercateo/sqs/utils/message/handling/MessageHandlingRunnable.java

```diff
@@ -81,7 +83,11 @@ public void run() {
 
     private void acknowledge(String messageId, Acknowledgment acknowledgment) {
         try {
-            acknowledgment.acknowledge().get();
+            try {
+                acknowledgment.acknowledge().get();
+            } catch (AmazonServiceException e) {
+                errorHandlingStrategy.handleAcknowledgeMessageException(e, message);
+            }
         } catch (Exception e) {
             log.error("could not acknowledge " + messageId, e);
```

Comment:
IMHO this should not have the same message like in the `DefaultErrorHandlingStrategy`
```suggestion
            log.error("failure during acknowledge " + messageId, e);
```